compiler/il/ILOpCodeProperties.hpp

```diff
@@ -2037,6 +2037,134 @@
+   /* .typeProperties       = */ ILTypeProp::Size_4 | ILTypeProp::Floating_Point,
+   /* .childProperties      = */ THREE_SAME_CHILD(TR::Float),
+   /* .swapChildrenOpCode   = */ TR::BadILOp,
+   /* .reverseBranchOpCode  = */ TR::BadILOp,
+   /* .booleanCompareOpCode = */ TR::BadILOp,
+   /* .ifCompareOpCode      = */ TR::BadILOp,
+   },
+
+   {
+   /* .opcode               = */ TR::fnmsub,
+   /* .name                 = */ "fnmsub",
+   /* .properties1          = */ 0,
+   /* .properties2          = */ ILProp2::ValueNumberShare | ILProp2::SupportedForPRE,
+   /* .properties3          = */ ILProp3::LikeUse,
+   /* .properties4          = */ 0,
+   /* .dataType             = */ TR::Float,
+   /* .typeProperties       = */ ILTypeProp::Size_4 | ILTypeProp::Floating_Point,
+   /* .childProperties      = */ THREE_SAME_CHILD(TR::Float),
+   /* .swapChildrenOpCode   = */ TR::BadILOp,
+   /* .reverseBranchOpCode  = */ TR::BadILOp,
+   /* .booleanCompareOpCode = */ TR::BadILOp,
+   /* .ifCompareOpCode      = */ TR::BadILOp,
+   },
+
+   {
+   /* .opcode               = */ TR::dmadd,
+   /* .name                 = */ "dmadd",
+   /* .properties1          = */ 0,
+   /* .properties2          = */ ILProp2::ValueNumberShare | ILProp2::SupportedForPRE,
+   /* .properties3          = */ ILProp3::LikeUse,
+   /* .properties4          = */ 0,
+   /* .dataType             = */ TR::Double,
+   /* .typeProperties       = */ ILTypeProp::Size_8 | ILTypeProp::Floating_Point,
+   /* .childProperties      = */ THREE_SAME_CHILD(TR::Double),
+   /* .swapChildrenOpCode   = */ TR::BadILOp,
+   /* .reverseBranchOpCode  = */ TR::BadILOp,
+   /* .booleanCompareOpCode = */ TR::BadILOp,
+   /* .ifCompareOpCode      = */ TR::BadILOp,
+   },
+
+   {
+   /* .opcode               = */ TR::dmsub,
+   /* .name                 = */ "dmsub",
+   /* .properties1          = */ 0,
+   /* .properties2          = */ ILProp2::ValueNumberShare | ILProp2::SupportedForPRE,
+   /* .properties3          = */ ILProp3::LikeUse,
+   /* .properties4          = */ 0,
+   /* .dataType             = */ TR::Double,
+   /* .typeProperties       = */ ILTypeProp::Size_8 | ILTypeProp::Floating_Point,
+   /* .childProperties      = */ THREE_SAME_CHILD(TR::Double),
+   /* .swapChildrenOpCode   = */ TR::BadILOp,
+   /* .reverseBranchOpCode  = */ TR::BadILOp,
+   /* .booleanCompareOpCode = */ TR::BadILOp,
+   /* .ifCompareOpCode      = */ TR::BadILOp,
+   },
+
+   {
+   /* .opcode               = */ TR::dnmadd,
+   /* .name                 = */ "dnmadd",
+   /* .properties1          = */ 0,
+   /* .properties2          = */ ILProp2::ValueNumberShare | ILProp2::SupportedForPRE,
+   /* .properties3          = */ ILProp3::LikeUse,
+   /* .properties4          = */ 0,
+   /* .dataType             = */ TR::Double,
+   /* .typeProperties       = */ ILTypeProp::Size_8 | ILTypeProp::Floating_Point,
+   /* .childProperties      = */ THREE_SAME_CHILD(ILChildProp::UnspecifiedChildType),
```

Comment:
Did this one get accidentally missed? :slightly_smiling_face:

Reply:
yeah. sorry about that. fixed now.